src/buildHTML.js

```diff
@@ -1231,6 +1247,19 @@ groupTypes.leftright = function(group, options) {
         ["minner", style.cls()], inner, options);
 };
 
+groupTypes.middle = function(group, options) {
+    var middleDelim;
+    if (group.value.value === ".") {
+        middleDelim = makeNullDelimiter(options, []);
+    } else {
+        middleDelim = delimiter.sizedDelim(
+            group.value.value, 1, options,
+            group.mode, []);
```

Comment:
You don't actually need to make this delimiter here, right? Shouldn't it always be overridden by the \left\right handler?